src/routes/spotify/desktop/attract/DVD.svelte

```diff
@@ -0,0 +1,51 @@
+<script lang="ts">
+  import { onMount, onDestroy } from "svelte";
+  import { Playlist } from "../../playlist.svelte";
+
+  let { playlist }: { playlist: ReturnType<typeof Playlist> } = $props();
+
+  const {
+    album: { art },
+    track: { title, artist },
+  } = playlist;
+
+  let x = $state(0);
```

Comment:
just a first pass, fresh from gpt

thinking it would be amazing to have two bouncing dvs, when the track changes we let the current track's go off the screen, and then the new track's album slides in

Reply:
funnnn, updated it per above!